### PR TITLE
Update README - how to trigger CI on external PRs

### PR DESCRIPTION
**Pull request by andrei-epure-sonarsource**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,8 +69,7 @@ _(they are difficult, can have many side effects and are less likely to be accep
 * We suggest you do not implement new rules unless they are already specified for C# and/or VB.NET on
 our [rules repository](https://jira.sonarsource.com/projects/RSPEC).
 
-Note: our CI runs only on PRs opened by sonarsourcers.
-After your PR is ready, we will cherry-pick your commits and open a PR to run the CI (keeping the attribution).
+Note: our CI does not get automatically triggered on the PRs of external contributors. A member of our team will do the code review and will trigger the CI on demand by adding a comment on the PR `/AzurePipelines run Sonar.Net` (see [Azure Pipelines Comment triggers docs](https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml#comment-triggers)).
```
Comment (pavel-mikula-sonarsource) [inline README.md]:
```suggestion
Note: Our CI does not get automatically triggered on the PRs from external contributors. A member of our team will do the code review and will trigger the CI on demand by adding a comment on the PR `/AzurePipelines run Sonar.Net` (see [Azure Pipelines Comment triggers docs](https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml#comment-triggers)).

```

**Comment by andrei-epure-sonarsource** [comment]
@pavel-mikula-sonarsource there's no need to manually trigger CI on our own PRs - it gets triggered automatically

**Comment by pavel-mikula-sonarsource** [comment]
@andrei-epure-sonarsource I was trying to restart it, because it got blocked on burger stage and there was no "re run" option at that step.

**Comment by pavel-mikula-sonarsource** [comment]
@andrei-epure-sonarsource Check is canceled, probably affected by changes in Pipelines. Please take a look at it.

**Pull request merged**